packages/weak-store/src/weakStore.js

```diff
@@ -12,26 +12,26 @@ import { insist } from '@agoric/insist';
  */
 function makeStore(keyName = 'key') {
   const wm = new WeakMap();
-  const insistKeyDoesNotExist = key =>
-    insist(!wm.has(key))([`${keyName} already registered`]);
-  const insistKeyExists = key =>
-    insist(wm.has(key))([`${keyName} not found: `, ''], key);
+  const assertKeyDoesNotExist = key =>
+    assert(!wm.has(key))([`${keyName} already registered`]);
```

Comment:
As above, I don't know how this could work.

Reply:
Yes, it doesn't yet work. Mark is working on what the solution might be. https://github.com/Agoric/agoric-sdk/pull/522#issuecomment-584346117

Reply:
This pattern is from when `insist()` was returning a template literal tag. Now that `details` is the template literal tag, and the details object is an argument to `assert`, the pattern needs to be redone in terms of that.